GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/StatusVerbTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using GVFS.FunctionalTests.FileSystemRunners;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+
+namespace GVFS.FunctionalTests.Tests.EnlistmentPerFixture
+{
+    [TestFixture]
+    public class StatusVerbTests : TestsWithEnlistmentPerFixture
+    {
+        private enum ExpectedReturnCode

```

Comment:
I don't see where this is used?

Reply:
Oops. It was needed for a helper function I had copied in an earlier version. Will delete.